lib/image/index.js

```diff
@@ -53,12 +73,58 @@ module.exports = class Image {
         );
     }
 
+    frame(area, opts = {}) {
+        area = this._scale(area, opts.scaleFactor);
+        const lineWidth = opts.scaleFactor,
+            color = '#feac88';
+
+        this._img.fill(
+            area.left,
+            area.top,
+            area.width,
+            lineWidth,
+            color
+        );
+        this._img.fill(
+            area.left,
+            area.top + area.height - lineWidth,
+            area.width,
+            lineWidth,
+            color
+        );
+        this._img.fill(
+            area.left,
+            area.top,
+            lineWidth,
+            area.height,
+            color
+        );
+        this._img.fill(
+            area.left + area.width - lineWidth,
+            area.top,
+            lineWidth,
+            area.height,
+            color
+        );
+    }
+
     join(newImage) {
         const imageSize = this.getSize();
+        const newIgnoreAreas = newImage.getIgnoreAreas();
+
         this._img
             .setSize(imageSize.width, imageSize.height + newImage.getSize().height)
             .insert(newImage._img, 0, imageSize.height);
 
+        newIgnoreAreas.forEach(({top, left, height, width}) => {
```

Comment:
I don't understand for what you write this code. Method `viewport.ignoreAreas` calling after image is already joined